addon/components/sticky-element.js

```diff
@@ -276,6 +276,9 @@ export default Component.extend({
   },
 
   updatePosition() {
+    if(this.get('isDestroyed') || this.get('isDestroying')) {
+      return false;
```

Comment:
As the function does not return anything, let's just `return` here. Otherwise could be confusing, as it would suggest the function has a return value.